src/library/scala/collection/View.scala

```diff
@@ -143,7 +143,7 @@ object View extends IterableFactory[View] {
   }
 
   /** A view that partitions an underlying collection into two views */
-  @SerialVersionUID(3L)
+  //@SerialVersionUID(3L)
```

Comment:
These warnings in the build should be resolved. Also, maybe the warning should sit behind a flag. `-Xlint:serial`? `-Ywarn-serial`? `-Ywarn-srsly`?